dashboards/vmalert.json

```diff
@@ -3236,6 +3236,213 @@
           ],
           "title": "Datapoints drop rate ($instance)",
           "type": "timeseries"
+        },
```

Comment:
shall we have panels for datasource and remote read metrics? Where do you think we should put them?

Btw, adding these metrics to the dashboard requires bumping vmalert version requirement here https://github.com/VictoriaMetrics/VictoriaMetrics/blob/c746ba154d4efbbd1a25cc98428f63092cd6659a/dashboards/vmalert.json#L83

Reply:
>shall we have panels for datasource and remote read metrics?

I don't think we need to put those metrics into dashboard since they don't display directly on anomalies like slow datasouce, pannels [Rules execution rate](https://github.com/VictoriaMetrics/VictoriaMetrics-enterprise/blob/64ed62ee1861133a2be05022873133c64c442c5e/dashboards/vmalert.json#L1023) and [Top groups avg evaluation duration](https://github.com/VictoriaMetrics/VictoriaMetrics-enterprise/blob/64ed62ee1861133a2be05022873133c64c442c5e/dashboards/vmalert.json#L920) are better than `vmalert_datasource_conns` and `vmalert_datasource_conn_bytes_read_total`[ as `rate(vmalert_datasource_conn_bytes_read_total)` decrease doesn't mean slow datasource].
And remoteRead metrics only have meaning when group starts.